lib/views/reporting/reviewing_card.dart

```diff
@@ -142,13 +152,56 @@ class ReportingCardViewState extends State<ReportingCardView> {
 
     return Column(
       children: [
-        Text(
-          widget.question,
-          style: const TextStyle(
-            color: CustomColors.primary,
-            fontSize: 18.0,
-            fontWeight: FontWeight.bold,
-          ),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            IconButton(
+              icon: const Icon(
+                Icons.arrow_back,
+                size: 28,
+                color: CustomColors.primary,
+              ),
+              onPressed: () {
```

Comment:
Code zoals in deze onpressed functie is heel lastig te begrijpen. Ik zou kijken of je dit misschien kan omschrijven en dan gebruik kan maken van elseif statements ipv else en daarin dan weer een if en dan weer een else